README.md

```diff
@@ -112,6 +112,26 @@ There are a few arguments that are required for cephci execution:
 #### Useful Arguments
 Some non-required arguments that we end up using a lot:
 * `--log level <level>` - set the log level that is output to stdout.
+* `--store` - stores the cluster configuration as snapshot under cephci/rerun directory
```

Comment:
`--store` could reword to mention such that
```
--store - store the deployed cluster information in a pickle file under cephci/rerun directory. 
```

Reply:
modified the statement.